samples/app/FilamentApp.cpp

```diff
@@ -329,13 +333,31 @@ void FilamentApp::run(const Config& config,SetupCallback setupCallback,
 
         // TODO: we need better timing or use SDL_GL_SetSwapInterval
         SDL_Delay(16);
+
         Renderer* renderer = window->getRenderer();
+
+        if (render) {
+            for (auto const& view : window->mViews) {
+                if (view.get() != window->mUiView) {
+                    render(mEngine, view->getView(), mScene, renderer);
+                }
+            }
+        }
+
         if (renderer->beginFrame(window->getSwapChain())) {
             for (auto const& view : window->mViews) {
                 renderer->render(view->getView());
             }
             renderer->endFrame();
         }
+
+        if (postRender) {
```

Comment:
I wouldn't bother with the `if` here since we're providing a default no-op function.